src/app/collective-rewards/components/BuildersCallToAction/BuildersCallToAction.tsx

```diff
@@ -0,0 +1,81 @@
+import { Header, Paragraph } from '@/components/TypographyNew'
+import { MetricsContainer } from '@/components/containers'
+import { Button } from '@/components/ButtonNew/Button'
+import { useAccount } from 'wagmi'
+import { zeroAddress } from 'viem'
+import { Banner } from '../Banner'
+import { BuildersDecorativeSquares } from '../DecorativeSquares'
+import { useBuilderContext } from '../../user'
+import { CallToActionCard } from '../CallToActionCard'
+import { ActiveBuilders } from '../ActiveBuilders'
+import { FC } from 'react'
+import { RewardsMetrics } from '../RewardsMetrics'
+import { useRouter } from 'next/navigation'
+
+const BuildersBanner = () => (
+  <Banner
+    imageSrc="/images/cta-banner-builders.png"
```

Comment:
Maybe we need to upload it to pinata

Reply:
Not sure, if this is the case, I think we can raise it to the DAO team and see what they think. IMHO this isn't a critical image for the protocol, so not sure if it makes sense to distribute it with IPFS